dialects/sqlite-3-18/src/main/kotlin/app/cash/sqldelight/dialects/sqlite_3_18/SqliteDialect.kt

```diff
@@ -0,0 +1,20 @@
+package app.cash.sqldelight.dialects.sqlite_3_18
+
+import app.cash.sqldelight.dialect.api.SqlDelightDialect
+import app.cash.sqldelight.dialect.api.TypeResolver
+import com.alecstrong.sql.psi.core.DialectPreset.SQLITE_3_18
+import com.squareup.kotlinpoet.ClassName
+
+/**
+ * A dialect for SQLite.
+ */
+class SqliteDialect : SqlDelightDialect {
+  override val driverType: ClassName get() = ClassName("app.cash.sqldelight.db", "SqlDriver")
+  override val cursorType: ClassName get() = ClassName("app.cash.sqldelight.db", "SqlCursor")
```

Comment:
```suggestion
  override val driverType: ClassName = ClassName("app.cash.sqldelight.db", "SqlDriver")
  override val cursorType: ClassName = ClassName("app.cash.sqldelight.db", "SqlCursor")
```
The ones above do not use `get() =`. This and the ones below do. Not sure if you want to be consistent. /shrug